### DISCUSSION TITLE
Setup Virtual Background - Issue

### DISCUSSION THREAD

**Discussion by OrangeHedgehog** [kounoike/obs-virtualbg#81] [Q&A]
Wanting to replace my Webcam background I installed OBS Studio 27.2.4 today, followed immediately by installing version 1.0.1 of this plugin (1.1.0 avoided due to seeing that some folks have had issues with it).  After following the instructions, the Webcam (Video Capture Device) preview turned blank.  I tried installing version 1.1.0 but then OBS Studio crashed on start. 

Copied the 1.0.1 `data `and `obs-plugins` folders back into the OBS Studio folder and OBS Studio starts again.  

Configuration steps:
Scene created
Added Webcam (_Video Capture Device_) to Sources.
Added static image to Sources.

With the _Video Capture Device_ selected, opened Filters.
Applied _Virtual Background Detector_ filter to the Audio/Video Filters.  Seems to be okay.
Applied _Virtual Background Renderer_ to the Effect Filters.  Video Capture Device preview goes blank.  This is even before clicking Close.
(These are the _only_ filters applied.)

_Virtual Background Detector_ settings are default, although I have tried changing the settings.
_Virtual Background Renderer_ setting is also default (_Blend_).  Tried _Render only_ but that did not help.

In Sources, _Video Capture Device_ is at the top, with the desired static image directly beneath it.  There are no other Sources.  I have tried reordering them, but that didn't help.

The _Video Capture Device_ properties are all default.

I have just one Scene.

In the Scources, the 'eye's are not crossed out, and the padlock is open.

Regarding the known conflict with _OBS Background removal plugin_, I have not installed that, so it's not that.

What is the likely issue, and how do I get the virtual background to work, please?

**Comment by TheXyL0X** [comment]
For this to work, you must add virtual background detector in the "Audio/video filter" section + Virtual background remover render in "Dilters of effects"

**Reply by OrangeHedgehog** [reply]
> For this to work, you must add virtual background detector in the "Audio/video filter" section + Virtual background remover render in "Dilters of effects"

I had them activated.

![image](https://user-images.githubusercontent.com/73525855/183283925-4dcf7de8-7043-4138-83ee-b20731032a7f.png)

**Comment by oliver4bsb** [comment]
to me it does nos show anything. its blank the page no settings nada.
